.gradient/workflows/training-pipeline.yaml

```
@@ -0,0 +1,225 @@
+
+workflows:
+  training-pipeline:
+    jobs:
+      - name: setup-environment
+        uses: script@v1
+        with:
+          script: |
+            # Install dependencies
+            pip install --upgrade pip
+            pip install -r requirements-workflow-prod.txt
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)